client/me/help/help-contact-closed/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import HelpContactClosedDetail from './detail';
+
+export default localize( ( props ) => {
+	const {	translate } = props;
+
+	return (
+		<div className="help-contact-closed">
+			<FormSectionHeading>{ translate( 'Limited Support This Week' ) }</FormSectionHeading>
+			<p>
+				{ translate(
+					'Private email and live chat support will be closed from %(closed_start_date)s through %(closed_end_date)s, ' +
+					'included. We will reopen private email support on %(email_open_date)s, and live chat support will be back as ' +
+					'usual on %(livechat_open_date)s.', {
+						args: {
+							closed_start_date: 'Wednesday, September 14th, 2016',
+							closed_end_date: 'Wednesdaty September 23rd, 2016',
+							email_open_date: 'Thursday, September 22nd',
+							livechat_open_date: 'Monday, September 26th',
+						}
+					}
+				) }
+			</p>
+			<p>
+				{ translate(
+					'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together ' +
+					'to work on improving our services, building new features, and learning how to better serve you, our users. ' +
+					'But never fear! if you need help in the meantime:'
+				) }
+			</p>
+			<HelpContactClosedDetail icon="help">
+				{ translate(
+					'The {{link}}forums{{/link}} remain open and staffed during this time, ' +
+					'and you can post a support request using the form below.', {
+						components: {
+							link: <a href="https://en.forums.wordpress.com/" target="_blank" rel="noopener noreferrer" />
```

Comment:
I think we should use just forums.wordpress.com, which'll forward people to their localized forums as needed.